docs/requirements.txt

```diff
@@ -1,7 +1,7 @@
 corner==2.2.1
 decorator>=4.2.1
 dill>=0.3.1.1
-dynesty==2.0.1
+dynesty==2.0.2
```

Comment:
Why do we need two identical requirements files?

Reply:
I could never figure out how to tell readthedocs to look elsewhere,  but it is also true that the docs build has specific requirements (e.g. the template) that the source doesnt.